src/openloco/ui/dropdown.cpp

```diff
@@ -18,6 +18,7 @@ namespace openloco::ui::dropdown
     static loco_global<uint32_t, 0x0113DC60> _dropdownDisabledItems;
     static loco_global<int16_t, 0x0113D84E> _dropdownHighlightedIndex;
     static loco_global<uint32_t, 0x0113DC64> _dropdownSelection;
+    static loco_global<std::uint8_t[32], 0x005046FA> _appropriateImageDropdownItemsPerRow;
```

Comment:
In IDA the size seems to be 33. Which is a odd, since the maximum in use seems to be 32...